src/compas_fab/ghpython/components/Cf_VisualizeRobot/code.py

```diff
@@ -3,11 +3,16 @@
 
 COMPAS FAB v0.27.0
 """
+import time
+
 from compas.artists import Artist
 from compas.geometry import Frame
 from compas.geometry import Transformation
-from compas_fab.robots import PlanningScene
 from ghpythonlib.componentbase import executingcomponent as component
+from scriptcontext import sticky as st
+
+from compas_fab.ghpython.components import create_id
```

Comment:
Maybe replace with `create_id` from `compas_ghpython`? one step towards retiring this one

Reply:
it would be nice, but it's still not released and it would bump the minimum requirement of compas for something rather trivial. What I could do thou, is `try..except` the import and keep the current function as fallback if it's not available.. not sure if it's worth it though

Reply:
ah that's a good point. nevermind then.